docs/howto/features/buckets.md

```diff
@@ -310,19 +320,60 @@ On AWS, we would need to set up [cross account S3 access](https://aws.amazon.com
+      each bucket, as some permissions apply to the bucket itself and some apply to the contents of
+      the bucket. We don't need to know the list of buckets here, the community champion can supply this.
+
+3. **Who**: 2i2c Engineering Team
+
+   Communicate to the community champion that they need to [attach a bucket policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html)
+   to the S3 buckets. You can use the following template.
 
    ```{note}
-   You can list as many buckets as you want, but each bucket needs two entries -
-   one with the `/*` and one without so both listing the bucket as well as fetching
-   data from it can work
+   To start the process, we need to attach policies to the appropriate S3 bucket to grant access
+   to the hub. I've generated these templates:
+
+   {{ JSON from step 2. Include multiple ones if we are doing this for multiple hubs }}
+
+   {{ if a staging hub is included }}
+   This includes access permissions for staging hub we use to test out changes to the main hub,
+   so those two can be in sync.
+   {{ end if }}
+
+   You need to fill in the list of S3 buckets you want granted access under the "Resource" section.
+   Note that there are two entries for each bucket.
+
+   This policy, as written, will grant readonly access to the buckets from the hub. Users
+   on the hub will be able to list objects and retrieve them. If you'd like to grant other
+   kinds of access, adjust the list under 'Actions' accordingly.
+
+   AWS has [some documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html)
+   on how to attach this bucket policy. Once attached, can you give us the full content of the
+   policy you attached? That would help us take next steps in granting access to these buckets.
+
+   Thanks!
    ```
 
-3. In the `.tfvars` file for the cluster hosting the hub, add `extra_iam_policy`
+4. **Who**: Community Champion
+
+   They attach the bucket policy, and get back to us with content of the policy they attached. This
+   helps us see the list of buckets that are authorized, as well as a record of what permissions
+   were actually granted.
+
+5. **Who**: 2i2c Engineering Team
+
+   In the `.tfvars` file for the cluster hosting the hub, add `extra_iam_policy`
    as a key to the hub under `hub_cloud_permissions`. This is used to set any additional
-   IAM permissions granted to the users of the hub. In this case, you should copy the
-   exact policy that was applied to the bucket in step 2, but remove the "Principal" key.
-   So it would look something like:
+   IAM permissions granted to the users of the hub. In this case, we use `s3:*` to allow
+   all S3 actions on the given list of buckets, so the exact permissions granted are
+   controlled by the bucket policy set in earlier steps. This allows the community to grant
+   more (or less) permissions in the future without any intervention on our part.
```

Comment:
I understand that this works, but I don't feel confident on what permissions we need to declare on the IAM Role already being granted permissions to work against the bucket in the bucket policy.

- Do we need to grant the bucket actions on the IAM role as well?
- Do we need to grant permissions to the IAM role to pay for access of this bucket if requestor pays is enabled on the bucket - if so, how is paying for bucket access configured? Is that perhaps under `s3:` something, and that is the reason we have needed `s3:*`?

If you are uncertain about this as well, I think it would be good to detail that this is done as a "ah this works" kind of solution as compared to "this is required" kind of solution.